packages/gp2-components/src/organisms/MainNavigation.tsx

```diff
@@ -2,10 +2,10 @@ import { drawerQuery, pixels } from '@asap-hub/react-components';
 import { gp2 } from '@asap-hub/routing';
 import { css } from '@emotion/react';
 import React from 'react';
-import DashboardIcon from '../icons/dashboard-icon';
-import ProjectIcon from '../icons/project-icon';
-import UsersIcon from '../icons/users-icon';
-import WorkingGroupIcon from '../icons/working-group-icon';
+import dashboardIcon from '../icons/dashboard-icon';
+import projectIcon from '../icons/project-icon';
+import usersIcon from '../icons/users-icon';
+import workingGroupIcon from '../icons/working-group-icon';
```

Comment:
```suggestion
import {
  dashboardIcon,
  projectIcon,
  usersIcon,
  workingGroupIcon,
}  from '../icons';
```
can we group all the icons imports, please?